index.bs

```diff
@@ -632,19 +632,281 @@ Issue(12): Propose control protocol for Remote
+and presentations with the consent of each origin (through their use of the
+API).  This is similar to cross-origin communication via {{Window/postMessage()}}
+with a target origin of `*`.  However, the Presentation API does not convey
+source origin information with each message.  Therefore, the Open Screen
+Protocol does not convey origin information between controllers and receivers.
+
+The [=presentation ID=] carries some protection against unrestricted
+cross-origin access; but, rigorous authentication of the parties connected by a
+{{PresentationConnection}} must be done at the application level.
+
+Open Screen Protocol Considerations {#security-privacy-questions}
 -----------------------------------
 
-Threat model {#security-threat}
+### Personally Identifiable Information & High-Value Data ### {#personally-identifiable-information}
+
+The following data exchanged by the protocol can be personally identifiable
+and/or high value data:
+
+1. Presentation URLs and availability results
+1. Presentation IDs
+1. Presentation connection IDs
+1. Presentation connection messages
+1. Remote playback URLs
+1. Remote playback commands and status messages
+
+Presentation display friendly names, model names, and capabilities, while not
+considered personally identifiable, is important to protect to preserve the
+integrity of the discovery and authentication process.
+
+The following data cannot be reasonably secured and should be considered public
+and untrusted data:
+
+1. IP addresses and ports used by the Open Screen Protocol.
+1. Data advertised through mDNS, including the certificate fingerprint and the 
+    metadata version.
+   
+### Cross Origin State Considerations ### {#cross-origin-state}
+
+Access to origin state across browsing sessions is possible through the
+Presentation API by reconnecting to a presentation that was started by a
+previous session. This scenario is addressed in
+[[PRESENTATION-API#cross-origin-access]].
+
+Presentation display availability and remote playback device availability are
+states that are available cross-origin depending on the user's network
+context.  Exposure of this data to the Web is also discussed in
+[[PRESENTATION-API#personally-identifiable-information]] and
+[[REMOTE-PLAYBACK#personally-identifiable-information]].
+
+### Origin Access to Other Devices ### {#origin-access-devices}
+
+By design, the Open Screen Protocol allows access to presentation displays and
+remote playback devices from the Web.  By implementing the protocol, these
+devices are knowingly making themselves available to the Web and should be
+designed accordingly.
+
+Below, we discuss mitigation steps to prevent malicious use of these devices.
+
+### Incognito Mode ### {#incognito-mode}
+
+The Open Screen Protocol does not distinguish between the user agent's normal
+browsing and incognito modes, and should operate identically regardless of which
+mode is in use.
```

Comment:
Will this always be true?  It seems like a very broad statement that might not always be true.

Reply:
Implementations could go beyond the spec to propagate information that would indicate if incognito is in use (even if it's a bad idea).  Added a caveat here.